themes/dark/templates/vue-components/show-selector.mako

```diff
@@ -56,11 +18,11 @@ select.select-show {
 Vue.component('show-selector', {
     template: '#show-selector-template',
     props: {
-        currentShowSlug: String
+        showSlug: String
     },
     data() {
         return {
-            selectedShowSlug: this.currentShowSlug
+            selectedShowSlug: this.showSlug
         };
     },
     computed: Object.assign(store.mapState(['config', 'shows']), {
```

Comment:
Just a question. Is this puex dependency injection?

Reply:
It's this https://vuex.vuejs.org/guide/state.html#the-mapstate-helper
Just using the global (window.)store object from store.js